modules.json

```diff
@@ -570,5 +570,17 @@
 			"usprop": "blockinfo|groups|editcount|registration|emailable|gender",
 			"format": "json"
         }
+    },
+	{
+        "filename": "get_alldeletedrevs",
+        "docstring": "Demo of `alldeletedrevisions` module: List the all deleted revisions from User",
+        "endpoint": "https://en.wikipedia.org/w/api.php",
+        "params": {
+            "action": "query",
+            "list": "alldeletedrevisions",
+            "adruser": "Mahesh",
```

Comment:
There's an inconsistency in the indentation of this schema

Reply:
@brendajerop  I have checked the identation and it is perfectly matching with the other file formats.

Reply:
![image](https://user-images.githubusercontent.com/20903830/66769412-c3a34e80-eebd-11e9-8583-d31699ee7d52.png)

Reply:
@brendajerop I'm using Pycharm editor and on the editor identation is perfectly correct. I don't understand why on github identation is wrong.
![image](https://user-images.githubusercontent.com/53967592/66771237-4dabe100-eed7-11e9-95aa-6ade7ee5d9fb.png)

Reply:
![image](https://user-images.githubusercontent.com/53967592/66793893-f7619100-ef1c-11e9-8f67-2e2e81257ac3.png)

Reply:
You may be mixing tabs and spaces and hence the inconsistency. Please, highlight your code in your editor to check :)

Reply:
thank you so much! @brendajerop :)